### ISSUE TITLE
TestNewJSRunnerWithCustomModule can't be run with `-count 2` or higher

### ISSUE THREAD

**Issue by mstoykov** [grafana/k6#1915]
```
$ go test -trimpath -run TestNewJSRunnerWithCustomModule -count 2
--- FAIL: TestNewJSRunnerWithCustomModule (0.00s)
panic: module already registered: k6/check [recovered]
        panic: module already registered: k6/check

goroutine 11 [running]:
testing.tRunner.func1.2(0xfcb140, 0xc0004f9730)
        testing/testing.go:1144 +0x332
testing.tRunner.func1(0xc000683080)
        testing/testing.go:1147 +0x4b6
panic(0xfcb140, 0xc0004f9730)
        runtime/panic.go:965 +0x1b9
github.com/loadimpact/k6/js/internal/modules.Register(0x1181a63, 0x8, 0x101eb60, 0xc0003995e0)
        github.com/loadimpact/k6/js/internal/modules/modules.go:60 +0x1b8
github.com/loadimpact/k6/js_test.TestNewJSRunnerWithCustomModule(0xc000683080)
        github.com/loadimpact/k6/js_test/init_and_modules_test.go:64 +0xb0
testing.tRunner(0xc000683080, 0x128e2c8)
        testing/testing.go:1194 +0xef
created by testing.(*T).Run
        testing/testing.go:1239 +0x2b3
exit status 2
FAIL    github.com/loadimpact/k6/js     0.028s
```
This happens because it tries to add a module to the global registry and that registry isn't cleaned between the multiple runs of `-count 2` or higher

**Comment by na--** [comment]
This should be fixed by https://github.com/loadimpact/k6/pull/1911, right?

**Comment by MStoykov** [comment]
There the module registry is still global so it will still blow up

**Comment by mstoykov** [comment]
Fixed in 54cc3cc39e460a19b41d7c31a306260bbb8c55f0
